relational_operators/CreateIndexOperator.cpp

```diff
@@ -28,11 +29,13 @@ bool CreateIndexOperator::getAllWorkOrders(WorkOrdersContainer *container,
                                            StorageManager *storage_manager,
                                            const tmb::client_id foreman_client_id,
                                            tmb::MessageBus *bus) {
+  database_->setStatus(CatalogDatabase::Status::kPendingIndexAdditions);
```

Comment:
No need to do so. Remove this and other related changes.

In fact, the status staff is only introduced for dropping relations in the distributed case where block deletions happen remotely.